source/documentation/quickstart/quickstart-guide.html.md

```diff
@@ -49,43 +43,50 @@ The following requirements are typical for small- to medium-sized installations.
 *   Static IP addresses for the oVirt Engine server and for each host server.
 *   DNS service that can resolve (forward and reverse) all the IP addresses.
 *   An existing DHCP server that can allocate network addresses for the virtual machines.
+*   If you need to use the storage from your hosts instead of using storage provided by SAN / NAS
```

Comment:
"If you need to use the storage from your hosts instead of using storage provided by SAN / NAS, consider moving to [Hyperconvergence](/documentation/gluster-hyperconverged/Gluster_Hyperconverged_Guide/).